Extension/src/LanguageServer/extension.ts

```diff
@@ -40,7 +40,7 @@ let activatedPreviously: PersistentWorkspaceState<boolean>;
 const insiderUpdateTimerInterval: number = 1000 * 60 * 60;
 let buildInfoCache: BuildInfo | null = null;
 const taskSourceStr: string = "C/C++";
-
+const cppInstallVsixStr: string = 'C/C++ install vsix: ';
```

Comment:
I was thinking at some point we'd want to prefix our log messages with `cpptools:`.  Perhaps this would be a good place to start.  It could be `cpptools: Install VSIX: ` if you want to have a separate category here.